production-marathon/marathon.env

```diff
@@ -1,16 +1,17 @@
 # The following properties MUST be configured before the Job Service can be deployed.
 
-export DOCKER_REGISTRY=NOT_SET_QA_index.docker.io/jobservice
+export DOCKER_REGISTRY=NOT_SET_DOCKER_REGISTRY
```

Comment:
Speaking with Andy there, the documentation and the standard set out for the environment variables is good.  The command for setting these and property substitution needs fixing.

I'm going to take a look at the command now and see if I can solve it. I suppose the README here if it's good to go can be merged in and the command at the bottom updated once a fix is found just simply update the documentation afterwards?